compiler/backend_inkwell/src/lib.rs

```diff
@@ -148,11 +154,17 @@ impl<'ctx> CodeGen<'ctx> {
     pub fn new(context: &'ctx Context, module_name: &str, mir: Arc<Mir>) -> Self {
         let module = context.create_module(module_name);
         let builder = context.create_builder();
+
+        let candy_value_type = context.opaque_struct_type("candy_value");
+        let candy_value_pointer_type = candy_value_type.ptr_type(AddressSpace::default());
+
         Self {
             context,
             module,
             builder,
             mir,
+            candy_value_pointer_type,
+            builtins: FxHashMap::default(),
             globals: HashMap::new(),
             locals: HashMap::new(),
             functions: HashMap::new(),
```

Comment:
```suggestion
            globals: FxHashMap::default(),
            locals: FxHashMap::default(),
            functions: FxHashMap::default(),
```

As above, please also change the initialisation of `unrepresented_ids`.